src/components/DynamicFormContainer/DynamicForm/DynamicForm.tsx

```diff
@@ -106,6 +133,15 @@ export const DynamicForm: FunctionComponent<DynamicFormProps> = ({
           }
         />
       )}
+      <label>Document Name</label>
+      <input
+        onChange={handleChangeFileName}
+        data-testid="file-name-input"
+        type="text"
+        aria-label="file-name-input"
+        value={newFileName}
+        className="border border-cloud-200 h-10 w-full rounded-lg px-3"
+      />
```

Comment:
Move this above the TransferableRecordForm like in the figma. and add a horizontal like to break it away from the form.

![Screenshot 2021-09-08 at 10 41 51 AM](https://user-images.githubusercontent.com/28627909/132437924-1dff8445-283f-4298-a846-b4ba2737dee3.png)